src/tests/unit/cpu/jit_kernel_test.cpp

```diff
@@ -159,6 +159,9 @@ struct jit_variable_test_kernel {
         }
 
         void generate() override {
+            init_load<float, float>(N);
```

Comment:
This actually brakes the concept. All load related logic is hided inside: `jit_kernel::load`. My proposal is to move emitters initialization as well: you just check if the map contains an emitter already or create it and put inside otherwise.